mpv.spec

```diff
@@ -95,15 +95,16 @@ waf configure \
     --disable-build-date \
     --enable-libmpv-shared \
     --enable-gpl3 \
-    --enable-sdl2
+    --enable-sdl2 \
+    --enable-encoding
```

Comment:
Encoding mode uses deprecated API. Encoding mode
will stop working/compiling at some point in the future,

(Note that the deprecations are not merged in FFmpeg yet, but they will
soon. They've been deprecated in Libav for a while now.)

2016-04-13 0:11 GMT+06:00 Miro Hrončok <notifications@github.com>:

> In mpv.spec
> <https://github.com/rpmfusion/mpv/pull/12#discussion_r59425247>:
>
> > @@ -95,15 +95,16 @@ waf configure \
> >      --disable-build-date \
> >      --enable-libmpv-shared \
> >      --enable-gpl3 \
> > -    --enable-sdl2
> > +    --enable-sdl2 \
> > +    --enable-encoding
>
> then why add the patch? I'm not sure I understand the question.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rpmfusion/mpv/pull/12/files/84afeb9ab76bc552bc6f30488e64e7e166953ada#r59425247>
>